src/components/XChangeButtonGrey.js

```diff
@@ -0,0 +1,24 @@
+import "./XChangeButtonGrey.css";
+import { Text } from "react-native";
+import Button from "@mui/material/Button";
+import { Link } from "react-router-dom";
+
+const s = {
+  fontStyle: "italic",
+};
+
+function XChangeButtonGrey(txt = "EMPTY", path = "", styl = s) {
```

Comment:
This can just be XChangeButton with colour parameter passed in

Reply:
changed